src/sentry/api/serializers/rest_framework/rule.py

```diff
@@ -92,6 +94,20 @@ def validate_actions(self, value):
             raise serializers.ValidationError(u"Must select an action")
         return value
 
+    def validate(self, attrs):
+        # XXX(meredith): For rules that have the Slack integration as an action
+        # we need to check if the channel_id needs to be looked up via an async task.
+        # If the "pending_save" attribute is set we want to bubble that up to the
+        # project_rule(_details) endpoints by setting it on attrs
+        actions = attrs.get("actions")
```

Comment:
Can remove the `if` block below with this:
```suggestion
        actions = attrs.get("actions", tuple())
```